### DISCUSSION TITLE
Unifi Gateways firmware 4.1.x incompatible with udm-iptv (UXG Lite)

### DISCUSSION THREAD

**Discussion by dmutsaers** [fabianishere/udm-iptv#358] [Q&A]
Has anyone else noticed issues with Unifi Gateways 4.1.2 and udm-iptv? If udm-iptv has been installed and configured the cpu on my UXG Lite udm-iptv runs at 100%. This results in high latency on my internet connection, dropping internet connection, and IPTV stutters or is impossble.

High CPU usage
https://community.ui.com/questions/High-CPU-usage/7b178f8c-6b2b-4fc4-835b-7b129ec52958

**Comment by latereaction** [comment]
Yep, same issue here - I've rolled back to the 4.0.x release and its fine again. monitoring htop, its the **ulogd** process - its spins up whenever the iptv box is on and receiving a channel

**Comment by dmutsaers** [comment]
Let's hope Ubiquiti doesn't release firmware 4.1.2 to the production channel before this issue has been fixed. Everyone with issues, please voice your concern here:
https://community.ui.com/releases/UniFi-Gateways-4-1-2/b77e3364-b598-43a2-a3e6-b887eb7bcbfd

**Comment by latereaction** [comment]
They have just released 4.1.3 to EA, anyone brave enough to try it? 
https://community.ui.com/releases/UniFi-Gateways-4-1-3/d27ea4cc-2638-4ce6-8643-603a1a55895c

[UXG-Lite](https://fw-download.ubnt.com/data/unifi-firmware/c70a-UXG-4.1.3-3bdadda1-d160-4451-90ad-731186915baf.bin)
[UXG-Max](https://fw-download.ubnt.com/data/unifi-firmware/8e4e-UXGB-4.1.3-f1c8b6ba-1ff9-4b16-b04a-5d87d49c23f3.bin)
[UXG-Pro](https://fw-download.ubnt.com/data/unifi-firmware/c896-UXGPROV2-4.1.3-5f47f20d-4e19-4253-a760-d0302e9f2447.bin)
[UXG-Enterprise](https://fw-download.ubnt.com/data/unifi-firmware/93e8-UXGENT-4.1.3-3701a982-a21c-4c40-a3f6-7cd6bb6a1795.bin)

**Reply by dmutsaers** [reply]
I just tried 4.1.3, the problem is still present (or even worse than in 4.1.2). Back to 4.0.20.

**Comment by dmutsaers** [comment]
Unifi's response: "We cannot assure stability on modified OS's, but I understand why you're adjusting the OS."

**Comment by dmutsaers** [comment]
Is anyone having the same problems with [Cloud Gateways firmware 4.1.9](https://community.ui.com/releases/UniFi-OS-Cloud-Gateways-4-1-9/10ac13d7-188a-4bc9-b83e-53239a9ca0e9) ?

**Comment by dmutsaers** [comment]
At least one user seems to have no problems with the latest firmware on a Cloud Gateway.

"Yes, I am running udm-iptv on my Cloud Gateway Ultra with v4.1.9
I see no issues personally (KPN)"

https://community.ui.com/releases/UniFi-OS-Cloud-Gateways-4-1-9/10ac13d7-188a-4bc9-b83e-53239a9ca0e9#comment/1f8dd580-6996-4837-a195-5900e323eea7

**Reply by Herbie5300** [reply]
Same over here with 1 exception: after manual update to Unifi OS 4.1.13 and Network version 9.0.108 last night IPTV gave me error STB-NMC-400. After reinstalling UDM-IPTV and uploading the udm-iptv.conf to the UCG MAX and a reboot of the UCG max and KPN Mediabox all worked fine again. I have just 1 box btw.

**Reply by dmutsaers** [reply]
That's to be expected, after a firmware update UDM-IPTV needs to be reinstalled.
I'm thinking that the UXG-Lite is not capable of IPTV/multicast, because a lack of resources (CPU).
Or is it because of the availability of more CPU cores (on more powerful devices than the UXG-Lite) that UDM-IPTV still causes a CPU being stuck at 99%, but users don't experience issues?

**Comment by dmutsaers** [comment]
[UniFi Gateways 4.1.8](https://community.ui.com/releases/UniFi-Gateways-4-1-8/5f9fefe3-3557-4f62-b754-fb4ff7980d5c) has been released, but IPTV issues persist.

**Comment by dmutsaers** [comment]
Could this be the issue?

> IPTV Streaming increases the system resource consumption. If you experience any performance issues, please disable the feature or upgrade to a more powerful Cloud Gateway.

**Comment by rogierlommers** [comment]
Just installed UnifiOS 4.1.13 on my Cloud Gateway Ultra. After upgrade I re-installed udm-iptv and everything works fine.

**Comment by dmutsaers** [comment]
thanks @rogierlommers for your reply.
It more and more begins to look like an issue with the UXG-Lite. (Or the non-cloud gateways)
